Comanche055/P76.agc

```diff
@@ -35,10 +35,10 @@
 #	TIME (TIG) AT WHICH THE OTHER VEHICLE VELOCITY WAS CHANGED BY EXECUTION OF A THRUSTING MANEUVER. THE
```

Comment:
✨ `--` instead of `-` on lines `32` and `33`

Reply:
Done